### PR TITLE
Typo (west instead of east and east instead of west)

### PR DESCRIPTION
**Pull request by OcelotWalrus**
When going to west, it prints `you can go east` and when you go east, it prints `you can go west` on the very first action.

By the way, cool game!

### CODE REVIEW

**Comment by lumbar527** [comment]
> By the way, cool game!

Thanks!

> By the way, what is the purpose of the `main data_file_converstion.py` script?

Basically, it's where I'm working on most stuff. But when I want to test fighting stuff, I just use `main.py` because moving around works!

**Comment by OcelotWalrus** [comment]
> Basically, it's where I'm working on most stuff. But when I want to test fighting stuff, I just use main.py because moving around works!

I see...

File: main.py [hunk 1]
```diff
@@ -95,7 +101,7 @@
 
     elif player["x"] == 1 and player["y"] == 0:
         which_direction = input(
-            "You can go west. There is a map on the ground. (type T to take the map, press M to view it when you have it.)")
+            "You can go east. There is a map on the ground. (type T to take the map, press M to view it when you have it.)")
```
Comment (lumbar527) [inline main.py:104]:
```suggestion
            "You can go west. There is a map on the ground. (type T to take the map, press M to view it when you have it.)")
```
For this game, west is negative.

File: main.py [hunk 2]
```diff
@@ -119,7 +125,7 @@
 
     elif player["x"] == -1 and player["y"] == 0:
         which_direction = input(
-            "You can go east. There is an axe on the ground. (type T to take the axe.)")
+            "You can go west. There is an axe on the ground. (type T to take the axe.)")
```
Comment (lumbar527) [inline main.py:128]:
```suggestion
            "You can go east. There is an axe on the ground. (type T to take the axe.)")
```
See above.

File: main.py [hunk 3]
```diff
@@ -168,7 +174,7 @@
 
     elif player["x"] == -1 and player["y"] == -1:
         which_direction = input(
-            "You can go east.")
+            "You can go west.")
```
Comment (lumbar527) [inline main.py:177]:
```suggestion
            "You can go east.")
```

File: main.py [hunk 4]
```diff
@@ -193,7 +199,7 @@
     elif player["x"] == 1 and player["y"] == -1:
         if fought_enemy == False:
             which_direction = input(
-                "You can go west. There is an enemy here...preparing to fight!")
+                "You can go east. There is an enemy here...preparing to fight!")
```
Comment (lumbar527) [inline main.py:202]:
```suggestion
                "You can go west. There is an enemy here...preparing to fight!")
```

**Review by lumbar527 — CHANGES_REQUESTED** [review]
I'll test this in a minute, but I think the title change is wrong.
If you want to explain that it is right, do so. (If you do, you also need to change the `.lower().startswith('w' or 'e')`).

**Comment by OcelotWalrus** [comment]
See:
image
When moving to east, its adds 1 to the x value. And when moving the west, it removes 1.

But here:
image
If the player x coordinates value are 1, it says it's west while it should say it's east because at the start, when choosing east, it adds 1 to x.

Same thing for under:
If the player x coordinates value are -1, its says it's east while it should say it's west because at the start, when choosing west, it removes 1 to x.

**Comment by OcelotWalrus** [comment]
Sory, it's just me who messed up when reading.

**Comment by lumbar527** [comment]
You need to add `import os` to the top of `main data_file_conversion.py`.

File: main data_file_conversion.py [hunk 5]
```diff
@@ -21,17 +21,22 @@
 create_save = input("Do you want to [o]pen saved game or create [n]ew game? ")
 
 if create_save.lower().startswith('n'):
-    save_name = input("Please name your save (make sure you add .yaml to the end): ")
+    enter_save_name = input("Please name your save (make sure you add .yaml to the end): ")
```
Comment (lumbar527) [inline main data_file_conversion.py:24]:
```suggestion
    enter_save_name = input("Please name your save: ")
```

**Review by lumbar527 — APPROVED** [review]
Looking good!

**Comment by lumbar527** [comment]
**Note:** Mostly I wouldn't bother with the `main.py` unless somethings broken. It will probably be removed once I get the map working.

**Comment by lumbar527** [comment]
If it's OK with you, I think I want to make it to Release by myself and then have other people contribute after that. But this is great and I'm merging once it's fixed up.

**Comment by OcelotWalrus** [comment]
Sure! It's just I saw this and I wanted to add a little of my contribution :)

**Comment by lumbar527** [comment]
I know a little bit, but not much.

**Comment by OcelotWalrus** [comment]
If you ever need help or have question don't hesitate to ask me :) I will be happy to help.

**Comment by lumbar527** [comment]
Thanks! I'll make an issue with you as an assignee if it's something not being discussed.

**Comment by OcelotWalrus** [comment]
Just one last thing, you should add a `.editorconfig` file.

**Comment by lumbar527** [comment]
How does that work? (you can make a PR for that too)

**Comment by lumbar527** [comment]
Visual Studio Code.

**Pull request merged**